TheWarWithin/RogueOutlaw.lua

```diff
@@ -1294,6 +1249,8 @@ spec:RegisterAbilities( {
             if pvptalent.take_your_cut.enabled then
                 applyBuff( "take_your_cut" )
             end
+
+            if talent.supercharger.enabled then addStack( "supercharged", nil, talent.supercharger.rank ) end
```

Comment:
Only forecast if we'll actually know which CP is charged.

Reply:
Not how it works anymore, it's a guaranteed action/effect now.